config/console.py

```diff
@@ -83,15 +96,18 @@ def add_console_setting(db, linenum, baud, flowcontrol, devicename):
 @click.argument('linenum', metavar='<line_number>', required=True, type=click.IntRange(0, 65535))
 def remove_console_setting(db, linenum):
     """Remove Console-related configuration tasks"""
-    config_db = db.cfgdb
+    config_db = ValidatedConfigDBConnector(db.cfgdb)
+    ctx = click.get_current_context()
 
     table = "CONSOLE_PORT"
 
     data = config_db.get_entry(table, linenum)
     if data:
-        config_db.mod_entry(table, linenum, None)
+        try:
+            config_db.set_entry(table, linenum, None)
+        except JsonPatchConflict as e:
```

Comment:
There're two exceptions: ValueError and JsonPatchConflict.
Why do you use JsonPatchConflict here and use ValueError for others?

Reply:
ValueError is thrown when attempting to set a field that already exists to a value that the Yang model does not allow. 

JsonPatchConflict is thrown when attempting to remove, or set a value to None, when the field does not already exist.